.circleci/config.yml

```diff
@@ -19,12 +19,12 @@ executors:
 
   builder:
     docker:
-      - image: omisegoimages/elixir-omg-builder:stable-20200410
+      - image: omisegoimages/elixir-omg-builder:dev-da1ecc8
     working_directory: ~/src
 
   builder_pg:
     docker:
-      - image: omisegoimages/elixir-omg-builder:stable-20200410
+      - image: omisegoimages/elixir-omg-builder:dev-da1ecc8
```

Comment:
are you planning to merge this pr when the corresponding pr in omisego-image is merged?